tutor/__init__.py

```diff
@@ -1,8 +1,10 @@
 from flask import Flask  
 from tutor.config import DevelopmentConfig
 from flask_sqlalchemy import SQLAlchemy
+from flask_bcrypt import Bcrypt
 
 
 app = Flask(__name__) 
 app.config.from_object(DevelopmentConfig)
 db = SQLAlchemy(app)
+bcrypt = Bcrypt(app)
```

Comment:
@mosesnagar 
I think it has to be declared here because the same bcrypt object has to be used to encrypt and decrypt. This means it is used by login and register controllers.